### ISSUE TITLE
Make command failed?

### ISSUE THREAD

**Issue by JustinKubicz** [ChetanXpro/nodejs-whisper#131]
Not sure what's going on here. I installed Cygwin with the make package and set it as a path. 

```
  await Whisper.nodewhisper(aFilePath, {
    modelName: "tiny.en",
    output: result + ".txt",
    verbose: true,
    removeWavFileAfterTranscription: false,
    withCuda: false,
    autoDownloadModelName: false,
    whisperOptions: {
      outputInText: true, //output in txt
      outputInVtt: false,
      outputInSrt: false,
      outputInCsv: false,
      translateToEnglish: false,
      timestamps_length: 20,
      wordTimestamps: false,
      splitOnWord: true,
    },
  });
};

```

I verified that aFilePath is valid, I'm pretty sure I did the setup correct. Admittedly I'm a novice. Here's the output...

```
[Nodejs-whisper] Error: [Nodejs-whisper] 'make' command failed. Please run 'make' command in /whisper.cpp directory.
[Nodejs-whisper] Error during processing: [Nodejs-whisper] 'make' command failed. Please run 'make' command in /whisper.cpp directory.

```

**Comment by Daxxboii** [comment]
bump!

**Comment by ChetanXpro** [comment]
@Daxxboii @JustinKubicz This issue should be fixed in PR [#136](https://github.com/ChetanXpro/nodejs-whisper/pull/136). Please let me know if you still face any issues.
